### PR TITLE
docs: replace TBD with a meaningful message

### PR DESCRIPTION
**Pull request by aleksei-korolev**
### Description

Replace "TBD" in dev's docu in chapters under construction with meaningful messages

### Type of change

- [X] This change requires a documentation update

### How Has This Been Tested?

I had read it out loud and clicked the links.

### Checklist

- [X] My code follows the [Coding Conventions](#coding-conventions)
- [X] I have performed a self-review of my code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by burnedikt — COMMENTED** [review]
You could also add a link to the file on github so someone with knowledge can create the documentation. Oftentimes seen in documentation of open source projects with a "Improve this Doc" Button. Also [supported by docFX](https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html#322-reserved-metadata)

File: .Documentation/articles/developer/TBD-02-editor-configuration.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Editor Configuration
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/v2.4.1/api/Innoactive.CreatorEditor.Configuration.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/developer/TBD-02-editor-configuration.md:5]:
```suggestion
This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.CreatorEditor.Configuration.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```

File: .Documentation/articles/developer/TBD-03-training-modes.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Training Modes
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/v2.4.1/api/Innoactive.Creator.Core.Configuration.Modes.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/developer/TBD-03-training-modes.md:5]:
```suggestion
This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Configuration.Modes.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```

File: .Documentation/articles/developer/TBD-04-configurators.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Configurators
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/v2.4.1/api/Innoactive.Creator.Core.Entity-1.html#Innoactive_Creator_Core_Entity_1_GetConfigurator) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/developer/TBD-04-configurators.md:5]:
```suggestion
This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Entity-1.html#Innoactive_Creator_Core_Entity_1_GetConfigurator) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```

File: .Documentation/articles/developer/TBD-05-mode-parameters.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
 # Mode Parameters
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/v2.4.1/api/Innoactive.Creator.Core.Configuration.Modes.ModeParameter-1.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/developer/TBD-05-mode-parameters.md:5]:
```suggestion
This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Configuration.Modes.ModeParameter-1.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```

File: .Documentation/articles/developer/TBD-06-localization.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
 # Localization
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/v2.4.1/api/Innoactive.Creator.Core.Internationalization.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (SimonTheSourcerer) [inline .Documentation/articles/developer/TBD-06-localization.md:5]:
```suggestion
This is a topic that we want to write about, but we just had no time for it yet. You can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Internationalization.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```

**Review by SimonTheSourcerer — CHANGES_REQUESTED** [review]

**Comment by SimonTheSourcerer** [comment]
I would like to just merge it after fixing the version thing because it's definitely much better than before.

**Comment by Gusinuhe** [comment]
When or where is it supposed to appear this `Improve this Doc` button?

**Comment by Maxsiom** [comment]
I think docFX does render it automatically.

**Comment by Maxsiom** [comment]
https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html#322-reserved-metadata
If you look there it is on the right site above the content.

**Comment by burnedikt** [comment]
> When or where is it supposed to appear this `Improve this Doc` button?

was just an idea, not a must. Enabling this should be as easy as defining `_gitContribute` in the `docfx.json` as indicated [here](https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html#322-reserved-metadata) but I am unsure if it will also require template changes as well to show that button / link. Can also be a separate PR.

**Comment by Gusinuhe** [comment]
I ask because I built the docs and I didn't find the button anywhere 🤔

**Comment by Maxsiom** [comment]
I put it into the meta data of our `docfx.json`. So I think it will now render it as soon as it is build. :)

**Review by SimonTheSourcerer — DISMISSED** [review]

File: .Documentation/articles/developer/TBD-06-localization.md [hunk 6]
```diff
@@ -1,3 +1,9 @@
...
 
-TBD
\ No newline at end of file
+Hi! You found a secret chapter!
+
+This is a topic that we want to write about, but we just had no time for it yet. 
+
+In the meantime you can find more information about the localization feature in the [Text to Speech Engine (TTS)](12-text-to-speech.md) and [Audio Behaviors](../innoactive-creator/default-behaviors.md#audioplay-audio-file) articles.
+
+Besides, you can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Internationalization.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of us developers will directly respond.
```
Comment (tomwim) [inline .Documentation/articles/developer/TBD-06-localization.md:9]:
```suggestion
Besides, you can take a look at our [API documentation](https://developers.innoactive.de/documentation/creator/latest/api/Innoactive.Creator.Core.Internationalization.html) instead and then ask your questions in our [community](http://innoactive.io/creator/community). One of our developers will directly respond.
```

**Review by tomwim — CHANGES_REQUESTED** [review]

**Review by tomwim — APPROVED** [review]

**Review by tomwim — APPROVED** [review]

**Pull request merged**